pyhive/tests/test_hive.py

```diff
@@ -18,7 +18,8 @@
 _HOST = 'localhost'
 
 
-@unittest.skipIf(sys.version_info.major == 3, 'Hive not yet supported on Python 3')
+@unittest.skipIf(sys.version_info.major == 3,
```

Comment:
It's a lot harder for me to merge & maintain stuff when there isn't test coverage :(